app/android/src/uk/co/lutraconsulting/PositionTrackingBroadcastMiddleware.java

```diff
@@ -46,7 +47,12 @@ public void registerBroadcastReceiver( Context context ) {
         intentFilter.addAction( TRACKING_ALIVE_STATUS_ACTION );
 
         // Set this as the receiver of POSITION_UPDATE and STATUS_MESSAGE intent actions
-        context.registerReceiver( serviceMessageReceiver, intentFilter );
+        if ( Build.VERSION.SDK_INT >= 34 ) { // Android 14 ( Upside Down Cake )
+            context.registerReceiver( serviceMessageReceiver, intentFilter, Context.RECEIVER_EXPORTED );
```

Comment:
Hmm, here we say that the broadcast is exported, but in the manifest we declared the service to not be exported. This works, but we might want to unify these (or keep it in mind) when finalizing https://github.com/MerginMaps/mobile/pull/3490